### PR TITLE
PS-632 : Changes responses of privileges apis

### PR DESCRIPTION
**Pull request by poojakarma**
## Summary by CodeRabbit

- **Bug Fixes**
  - Improved error and success response handling for privilege and role management.

- **Refactor**
  - Standardized API response handling across privilege and role services for consistency.
  - Replaced hardcoded API IDs with constants for better maintainability.
  - Updated error messages and status codes for clarity.

### CODE REVIEW

File: src/adapters/postgres/user-adapter.ts [hunk 1]
```diff
@@ -914,25 +914,26 @@ export class PostgresUserService {
...
-        return APIResponse.error(response, apiId, "Internal Server Error", `Error : ${e?.response?.data.error}`, String(HttpStatus.INTERNAL_SERVER_ERROR));
-      }
+          'Authorization': `Bearer ${token}`
+        }
+      });
+
+
+      // return new SuccessResponse({
+      //   statusCode: HttpStatus.OK,
+      //   message: "User and related entries deleted Successfully.",
```
Comment (vijaykhollam) [inline src/adapters/postgres/user-adapter.ts:924]: @poojakarma Why commented code

File: src/adapters/postgres/rbac/privilege-adapter.ts [hunk 2]
```diff
@@ -53,19 +57,16 @@ export class PostgresPrivilegeService {
         privileges.push(new PrivilegeResponseDto(response));
       }
     } catch (e) {
-      return new ErrorResponseTypeOrm({
-        statusCode: HttpStatus.INTERNAL_SERVER_ERROR,
-        errorMessage: e.message || "Internal server error",
-      });
+      return APIResponse.error(response, apiId, "Internal Server Error", `Error is ${e}`, HttpStatus.INTERNAL_SERVER_ERROR);
```
Comment (vaivk369) [inline src/adapters/postgres/rbac/privilege-adapter.ts:60]: @poojakarma 
       const errorMessage = e.message || "Internal server error";
      return APIResponse.error(response, apiId, "Internal Server Error", errorMessage, HttpStatus.INTERNAL_SERVER_ERROR);

Comment (vaivk369) [reply src/adapters/postgres/rbac/privilege-adapter.ts:60]: @poojakarma Follow this for all the error messages thrown in this file

Comment (poojakarma) [reply src/adapters/postgres/rbac/privilege-adapter.ts:60]: ok mam

**Pull request merged**